Formula/zpython.rb

```diff
@@ -1,20 +1,3 @@
-class Zsh5Requirement < Requirement
-  default_formula "zsh"
-  fatal true
-
-  satisfy :build_env => false do
-    begin
-      `zsh --version`[/zsh (\d)/, 1] == "5"
-    rescue
-      false
-    end
-  end
-
-  def message
-    "Zsh 5.x is required to install. Consider `brew install zsh`."
-  end
-end
-
 class Zpython < Formula
   desc "Embeds a Python interpreter into zsh"
   homepage "https://bitbucket.org/ZyX_I/zsh"
```

Comment:
Hey there, seems like this site is down for the count, can we replace it with http://www.zsh.org/?

Reply:
It's a network fluke on CI, works fine here.

Reply:
Ah, never mind, it seems like it was actually something with the BitBucket user -- I tried connecting earlier, but I got a 404 on my own end.  Never mind!